sources/packages/backend/libs/services/src/disbursement-overaward/disbursement-overaward.service.ts

```diff
@@ -70,4 +75,58 @@ export class DisbursementOverawardService {
     }
     return result;
   }
+
+  /**
+   * Get all overawards which belong to a student.
+   * @param studentId student.
+   * @returns overaward details of a student.
+   */
+  async getOverawardsByStudent(
+    studentId: number,
+  ): Promise<DisbursementOveraward[]> {
+    return this.disbursementOverawardRepo.find({
+      select: {
+        createdAt: true,
+        originType: true,
+        overawardValue: true,
+        disbursementValueCode: true,
+        creator: { firstName: true, lastName: true },
```

Comment:
Not part of the PR but we have been not using `creator` for this type of auditing purposes. I guess that this is a miss in the same way that we missed adding the notes here also. Thoughts?

Reply:
I assume you are saying that we must have an exclusive column to identify the user who added the overaward/note right?